app/src/ui/merge-conflicts/merge-conflicts-dialog.tsx

```diff
@@ -165,9 +172,15 @@ export class MergeConflictsDialog extends React.Component<
       case AppFileStatus.Resolved:
         return this.renderResolvedFile(file.path)
       case AppFileStatus.Conflicted:
+        if (file.conflictStatus === null) {
+          throw new Error(
```

Comment:
yeah i agree this isn't ideal, but I think to remove it we'd need to restructure status parsing/construction more. i think the solution is to move `conflictStatus` out of `WorkingDirectoryFileChange` and into `AppFileStatus`, but then `AppFileStatus` would have to be more than just an enum. anyways, that's a long way of saying i think this is fine for now.